packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -171,4 +171,6 @@ exports.config = {
     ui: 'bdd',
     timeout: 60000,
   },
+
+  serviceOptions: {},
```

Comment:
do we need to define this if we aren't setting any custom service options at the config level?

Reply:
+1 to removing this if there are no options